src/util.js

```diff
@@ -13,17 +11,19 @@ export function mapActions(actions, store) {
 export function select(properties) {
 	if (typeof properties==='string') properties = properties.split(/\s*,\s*/);
 	return state => {
-		let selected = {};
-		for (let i=0; i<properties.length; i++) {
-			selected[properties[i]] = state[properties[i]];
-		}
+		const selected = {};
+		let i = properties.length;
+		while (i-- > 0) selected[properties[i]] = state[properties[i]];
 		return selected;
 	};
 }
 
 
-// Lighter Object.assign stand-in
-export function assign(obj, props) {
-	for (let i in props) obj[i] = props[i];
-	return obj;
+// Lighter Object.assign clone
+export function assign(_) {
```

Comment:
multi object support

Reply:
This is expensive: now `assign()` has variable arity and is unlikely to be optimized.

Reply:
It saves bytes compared to its uses.
Also, I'm pretty sure that this is a safe use for optimizations. See https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#what-is-safe-arguments-usage

Reply:
On this note, why not just use `Object.assign`? Support is extremely good: http://kangax.github.io/compat-table/es6/#test-Object_static_methods

Reply:
how are you checking the byte savings? we only care about gzipped output size, not the source size.

Reply:
Would you "object" (pun intended) to documenting a need for a polyfill for older (IE <= 11) browser support? Could rip this out altogether.
If not:
I based my saved bytes comment off the fact that assign doesn't need multiple invocations for combining multiple objects, as is its use case in the code. I could test which net-size is smaller between the two ways.